global.json

```diff
@@ -1,6 +1,6 @@
 {
   "tools": {
-    "dotnet": "9.0.100-rc.1.24452.12"
+    "dotnet": "8.0.403"
```

Comment:
The `"tools"` element in `global.json` doesn't do anything locally, so I'd revert this. This only applies for CI, as something dotnet/arcade invented.

Reply:
Planned to remove that too. 

Simply copypasta so I can work on real problem.